src/edtr-io/plugins/page-layout/index.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  EditorPlugin,
+  number,
+  object,
+  EditorPluginProps,
+} from '@edtr-io/plugin'
+
+import { editorContent } from '../types/common/common'
+import { PageLayoutEditor } from './editor'
+
+export const pageLayoutState = object({
+  widthPercent: number(), // first column defined second
```

Comment:
I would use an enum with 3 values here. Then I would add a function converting this enum into a percentage number.